decidim-templates/app/views/decidim/templates/admin/questionnaire_templates/edit_questions.erb

```diff
@@ -15,7 +15,7 @@
 
 <div class="item__edit item__edit-1col">
   <div class="item__edit-form">
-    <%= decidim_form_for(@form, url: update_questions_questionnaire_template_path(questionnaire), method: :patch, html: { class: "form-defaults form edit_questions_survey" }) do |form| %>
+    <%= decidim_form_for(@form, url: update_questions_questionnaire_template_path(template), method: :patch, html: { class: "form-defaults form edit_questions_survey" }) do |form| %>
```

Comment:
The templates module needs a test where a full creation of a survey template is created. This should check this line

Reply:
Done